crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -267,11 +267,12 @@ en:
     service:
       name_exists: 'Name already exists. Please choose another.'
       too_short: 'Invalid name.  Name must have some characters.'
-      illegal_chars: 'Invalid service name %{name}. Please limit name to [A-z0-9_] without spaces.'
-      illegal_name: 'Illegal proposal name %{name}. Please choose another proposal name.'
+      illegal_chars: 'Please limit name to [A-z0-9_] without spaces.'
+      illegal_name: 'Please choose another proposal name.'
```

Comment:
Might be worth clarifying why, as this is totally unclear from this message the name is forbidden.